### PR TITLE
MNT: This shows all the PEP 8 violations, except explicitly ignored

### LINKED ISSUE #400: flake8: Enable checking for F401 and others
Currently only select violation codes are checked and F401 is not one of them, causing unused imports in modules.

### PR DESCRIPTION
**Pull request by pllim**
Fix #400 and a little more.

### CODE REVIEW

**Comment by pllim** [comment]
Only 455...

**Comment by pllim** [comment]
@duytnguyendtn , probably worth half a sprint to clean this up. Having an effective check, especially for imports and variables can reduce chances of bugs. FYI

**Comment by ibusko** [comment]
I don't understand why the specific selections where removed, in favor of the default.

File: setup.cfg [hunk 1]
```diff
@@ -70,7 +70,8 @@ addopts = --doctest-rst
 
 [flake8]
 max-line-length = 100
-select = E101,W191,W291,W292,W293,W391,E111,E112,E113,E502,E722,E901,E902
+# E402: Module level import not at top of file
+ignore = E402
```
Comment (pllim) [inline setup.cfg:74]: @ibusko , currently it only checks for a few via `select`. What I did here is to have it check for everything except `ignore`. Does that clear up your confusion?

**Review by ibusko — APPROVED** [review]

**Comment by pllim** [comment]
And to clarify: This is still a draft PR, as the existing violations need to be addressed (whether fixed or conciously put under `ignore`).

**Comment by pllim** [comment]
Closing in favor of #425

**Pull request closed without merging**